ndcube/tests/test_cube_utils.py

```diff
@@ -1,69 +1,107 @@
 # -*- coding: utf-8 -*-
 import pytest
-from ndcube import cube_utils as cu
 import numpy as np
-from ndcube.wcs_util import WCS
-from astropy import units as u
+
+from ndcube import utils
 
 ht = {
-    'CTYPE1': 'HPLT-TAN', 'CUNIT1': 'deg', 'CDELT1': 0.5, 'CRPIX1': 0, 'CRVAL1': 0, 'NAXIS1': 2,
-    'CTYPE2': 'WAVE    ', 'CUNIT2': 'Angstrom', 'CDELT2': 0.2, 'CRPIX2': 0, 'CRVAL2': 0, 'NAXIS2': 3,
-    'CTYPE3': 'TIME    ', 'CUNIT3': 'min', 'CDELT3': 0.4, 'CRPIX3': 0, 'CRVAL3': 0, 'NAXIS3': 4
+    'CTYPE1': 'HPLT-TAN',
+    'CUNIT1': 'deg',
+    'CDELT1': 0.5,
+    'CRPIX1': 0,
+    'CRVAL1': 0,
+    'NAXIS1': 2,
+    'CTYPE2': 'WAVE    ',
+    'CUNIT2': 'Angstrom',
+    'CDELT2': 0.2,
+    'CRPIX2': 0,
+    'CRVAL2': 0,
+    'NAXIS2': 3,
+    'CTYPE3': 'TIME    ',
+    'CUNIT3': 'min',
+    'CDELT3': 0.4,
+    'CRPIX3': 0,
+    'CRVAL3': 0,
+    'NAXIS3': 4
 }
-wt = WCS(header=ht, naxis=3)
+wt = utils.wcs.WCS(header=ht, naxis=3)
 
 hm = {
-    'CTYPE1': 'WAVE    ', 'CUNIT1': 'Angstrom', 'CDELT1': 0.2, 'CRPIX1': 0, 'CRVAL1': 10, 'NAXIS1': 4,
-    'CTYPE2': 'HPLT-TAN', 'CUNIT2': 'deg', 'CDELT2': 0.5, 'CRPIX2': 2, 'CRVAL2': 0.5, 'NAXIS2': 3,
-    'CTYPE3': 'HPLN-TAN', 'CUNIT3': 'deg', 'CDELT3': 0.4, 'CRPIX3': 2, 'CRVAL3': 1, 'NAXIS3': 2,
+    'CTYPE1': 'WAVE    ',
+    'CUNIT1': 'Angstrom',
+    'CDELT1': 0.2,
+    'CRPIX1': 0,
+    'CRVAL1': 10,
+    'NAXIS1': 4,
+    'CTYPE2': 'HPLT-TAN',
+    'CUNIT2': 'deg',
+    'CDELT2': 0.5,
+    'CRPIX2': 2,
+    'CRVAL2': 0.5,
+    'NAXIS2': 3,
+    'CTYPE3': 'HPLN-TAN',
+    'CUNIT3': 'deg',
+    'CDELT3': 0.4,
+    'CRPIX3': 2,
+    'CRVAL3': 1,
+    'NAXIS3': 2,
 }
-wm = WCS(header=hm, naxis=3)
+wm = utils.wcs.WCS(header=hm, naxis=3)
 
 data = np.array([[[1, 2, 3, 4], [2, 4, 5, 3], [0, -1, 2, 3]],
                  [[2, 4, 5, 1], [10, 5, 2, 2], [10, 3, 3, 0]]])
 
 
 def test_select_order():
-    lists = [['TIME', 'WAVE', 'HPLT-TAN', 'HPLN-TAN'],
-             ['WAVE', 'HPLT-TAN', 'UTC', 'HPLN-TAN'],
-             ['HPLT-TAN', 'TIME', 'HPLN-TAN'],
-             ['HPLT-TAN', 'DEC--TAN', 'WAVE'],
-             [],
-             ['UTC', 'TIME', 'WAVE', 'HPLT-TAN']]
+    lists = [['TIME', 'WAVE', 'HPLT-TAN',
+              'HPLN-TAN'], ['WAVE', 'HPLT-TAN', 'UTC',
+                            'HPLN-TAN'], ['HPLT-TAN', 'TIME', 'HPLN-TAN'],
+             ['HPLT-TAN', 'DEC--TAN',
+              'WAVE'], [], ['UTC', 'TIME', 'WAVE', 'HPLT-TAN']]
```

Comment:
Why is the nice readable layout changed here?

Reply:
I auto-layouted it, but this is not nice!

Reply:
What's auto-layout??

Reply:
Is that the same reason the parameterize (input, expected) decorator has exploded?

Reply:
https://github.com/google/yapf

yeah, I will give it a tidy

Reply:
I’ve merged this PR and started work on reorganising the testing modules and increasing coverage. Let me open a new PR with my progress so far and you can tidy it there.

Reply:
See #49